contracts/anchors/bridged/LinkableCompTokenAnchorPoseidon2.sol

```diff
@@ -6,13 +6,15 @@
 pragma solidity ^0.8.0;
 
 import "../../tokens/TokenWrapper.sol";
+import "../../interfaces/ITokenWrapper.sol";
 import "./LinkableAnchorPoseidon2.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
 contract LinkableCompTokenAnchorPoseidon2 is LinkableAnchorPoseidon2 {
   using SafeERC20 for IERC20;
   address public immutable token;
+  bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
```

Comment:
We want to avoid this.

Reply:
Avoid it as it is repetitive and should only be relevant to the TokenWrapper? I want to make sure I understand why to avoid it so I dont make the same mistake going forward